packages/client/tests/functional/extensions/defineExtension.ts

```diff
@@ -619,14 +619,12 @@ testMatrix.setupTestSuite(() => {
       })
       expectTypeOf<typeof _upsert>().toEqualTypeOf<typeof upsert>()
 
-      // @ts-test-if: provider === 'mongodb'
       const _findRaw = xprisma.user._findRaw({})
       // @ts-test-if: provider === 'mongodb'
       const findRaw = await prisma.user.findRaw({})
       // @ts-test-if: provider === 'mongodb'
       expectTypeOf<typeof _findRaw>().toEqualTypeOf<typeof findRaw>()
 
-      // @ts-test-if: provider === 'mongodb'
       const _aggregateRaw = xprisma.user._aggregateRaw({})
```

Comment:
Side-effect of the change, the input turned from `never` to `any` so skipping this type check does not make sense anymore.